justifi-tech/web-component-library
# Upgrade react related packages

Upgrade React and React peer dependencies for compatibility with new React 18 apps.

Links
-----



Developer considerations
--------------

  - If this is a MAJOR version, what are the breaking API changes? Was anything added to the API?
  - If this is a MINOR version, what feature was added? Why would consumers want this new version?
  - If this is a PATCH version, what was fixed?


Steps for Testing/QA
--------------------
Since these are just auto generated we just need to check that the build is still successful.
